### PR TITLE
Suse

### PR DESCRIPTION
**Pull request by lobiCode**
Add support for `openSUSE` and `SLES`.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -8,7 +8,8 @@ CI_COMMIT     ?= dev
 GIT_COMMIT    := $(shell git rev-parse --short HEAD || echo $(CI_COMMIT))
 VERSION       ?= $(shell cat ./VERSION)
 FLAGS         := "-X main.GitCommit=$(GIT_COMMIT) -X main.Version=$(VERSION)"
-DISTRIBUTIONS := ubuntu debian rhel centos
+DISTRIBUTIONS := ubuntu debian rhel centos opensuse sles
+RPM 	      := sles opensuse
```
Comment (dominikznidar) [inline Makefile]: Please mind the column formatting.

File: host_dflt.go [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// +build ubuntu debian
```
Comment (dominikznidar) [inline host_dflt.go]: Does `dflt` stand for `default`? If so, lets just use `default`.

File: host_dflt_test.go [hunk 3]
```diff
@@ -1,4 +1,4 @@
-// +build ubuntu debian
```
Comment (dominikznidar) [inline host_dflt_test.go]: Same `dflt` to `default` comment here.

File: host_rhel.go [hunk 4]
```diff
@@ -1,4 +1,4 @@
-// +build rhel centos
+// +build rhel centos debian
```
Comment (dominikznidar) [inline host_rhel.go:1]: It's a bit confusing `debian` is inside `host_rhel` file. This looks like a common generator for "What was ... thinking here ..." comments in the future :)

**Review by dominikznidar — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (lobiCode) [reply Makefile]: Line `12` has been removed. We don't need it

File: host_dflt.go (hunk 2 above)
Comment (lobiCode) [reply host_dflt.go]: `dflt` has been changed to `default`

File: host_dflt_test.go (hunk 3 above)
Comment (lobiCode) [reply host_dflt_test.go]: `dflt` has been changed to `default`

File: host_rhel.go (hunk 4 above)
Comment (dominikznidar) [reply host_rhel.go:1]: Lets just leave it for now.

**Review by dominikznidar — APPROVED** [review]

**Pull request merged**